app/controllers/galleries_controller.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 class GalleriesController < ApplicationController
+  helper_method :documents
```

Comment:
True, true.

A refactor would be worthwhile here I reckon, and should not be too tricky, something like: https://github.com/europeana/europeana-portal-collections/commit/4e1bea340cc3fcba630a850f6d878e90e3f9cb52